README.md

```diff
@@ -12,12 +12,18 @@ As Hypervisor the following systems are supported:
 
 - List existing snapshots of a virtual machine.
 - Create a snapshot of a virtual machine.
-- Revert existing virtual machine to a previously created snapshot.
+- Revert existing virtual machines to a previously created snapshot.
 - Remove existing snapshots of a virtual machine.
 
 ## Installation
 
-See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin) for how to install Foreman plugins
+This plugin installation is not supported by the foreman-installer, you need to install it manually:
```

Comment:
You can install Foreman Snapshot Management plugin by installing the package and then running `foreman-installer` on your Foreman server:

Reply:
Much better phrasing, but I double-checked and found that this plugin actually IS supported by the installer :see_no_evil: